### ISSUE TITLE
最终输出的视频缺少躯干

### ISSUE THREAD

**Issue by Guoxni** [yerfor/GeneFace#257]
最终输出的may视频只有头部没有躯干

val: Smooth head trajectory (rotation and translation) with a window size of 7
Setting up [LPIPS] perceptual loss: trunk [alex], v[0.1], spatial [off]
Loading model from: /root/miniconda3/envs/geneface/lib/python3.9/site-packages/lpips/weights/v0.1/alex.pth
| load 'model' from 'checkpoints/May/lm3d_radnerf/model_ckpt_steps_250000.ckpt'.
Loaded Head Model from checkpoints/May/lm3d_radnerf
Loaded state_dict of Head Model to the RADNeRFTorso Model
| load 'model' from 'checkpoints/May/lm3d_postnet_sync/model_ckpt_steps_4000.ckpt'.
The tmp imge dir is infer_out/May/pred_video/tmp_imgs/zozo_radnerf_torso_smo.
NeRF is rendering frames...: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████| 272/272 [00:24<00:00, 10.91it/s]
infer_out/May/pred_video/tmp_imgs/zozo_radnerf_torso_smo data/raw/val_wavs/zozo_16k.wav infer_out/May/pred_video/zozo_radnerf_torso_smo.mp4
The synthesized video is saved at infer_out/May/pred_video/zozo_radnerf_torso_smo.mp4

**Comment by Guoxni** [comment]
autodl-tmp/GeneFace-main/infer_out/May/pred_video/tmp_imgs/zozo_radnerf_torso_smo中的图片也是只有头部没有躯干
请问这是什么原因呢

**Comment by feipiefei** [comment]
我输的只有图片没有视频是什么原因呢～

**Comment by feipiefei** [comment]
<img width="1004" alt="image" src="https://github.com/yerfor/GeneFace/assets/40300259/15dedbde-a7a9-424e-8b23-40ebae6341dc">
<img width="1095" alt="image" src="https://github.com/yerfor/GeneFace/assets/40300259/8c17f25c-4ae7-4821-83f8-562c001a84b5">

**Comment by Guoxni** [comment]
谢谢帮助！修改work_dir之后可以生成躯干了！
---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年01月10日 11:36 |
| 收件人 | yerfor/GeneFace ***@***.***> |
| 抄送人 | Guoxni ***@***.***>,
Author ***@***.***> |
| 主题 | Re: [yerfor/GeneFace] 最终输出的视频缺少躯干 (Issue #257) |

image.png (view on web)image.png (view on web)
